event.go

```diff
@@ -235,6 +235,30 @@ func (e *Event) Strs(key string, vals []string) *Event {
 	return e
 }
 
+// Stringer adds the field key with val as a string to the *Event context.
+// Use Stringer if possible, because it creates string value in lazy mode.
+func (e *Event) Stringer(key string, val fmt.Stringer) *Event {
+	if e == nil {
+		return e
+	}
+	if val == nil {
+		e.buf = enc.AppendString(enc.AppendKey(e.buf, key), "nil")
```

Comment:
You can use e.buf = enc.AppendInterface(enc.AppendKey(e.buf, key), nil) to achieve that